### ISSUE TITLE
Weird stuff happens when you holster a gun with a no-drop implant active

### ISSUE THREAD

**Issue by LittleBigKid2000** [ParadiseSS13/Paradise#4313]
I unholstered my .38, activated my no-drop implant, played some solo russian roulette with it, and eventually shot myself but didn't die since it was only rubber bullets. I holstered my gun, and that's when weird stuff happened. The gun was both in my right hand and in the holster. I unholstered it, and the gun was now in my left hand, but at the same time in my right hand, but invisible. I took off the holster, and it apparently already had a gun in it. Even if the no-drop implant was deactivated, none of these guns could be thrown, dropped, etc.
It might be worth noting that I was an IPC

TL;DR: Guns can be holstered with a no-drop implant active, but the gun stays in the hand, making a single gun be at multiple places at once

**Comment by Fox-McCloud** [comment]
Problem is you shouldn't be able to get something with NO DROP out of your hands.
